frontend/src/api/wordEditorAPI.ts

```diff
@@ -157,5 +157,34 @@ export const wordEditorAPI: EditorAPI = {
 				reject(error);
 			}
 		});
+	},
+
+	// Get the text of the current paragraph and the text of all paragraphs in the document
+	GetParagraphTexts(): Promise<{ curParagraphIndex: number; newParagraphTexts: string[] }> {
```

Comment:
function name is inconsistent with other functions here. But see my top-level comment: we might not even need this function.